Makefile

```diff
@@ -4,16 +4,18 @@ fmt:
 	go fmt ./...
 
 install: fmt
+	touch consensus/blocknode.go
 	go install -tags=dev ./...
 
 clean:
 	rm -rf hostdir release whitepaper.aux whitepaper.log whitepaper.pdf         \
 		sia.wallet sia/test.wallet sia/hostdir* sia/renterDownload
 
-test: clean
+test: clean fmt
+	touch consensus/blocknode.go
```

Comment:
look at the [mkvsynth Makefile](https://github.com/NebulousLabs/mkvsynth/blob/master/Makefile#L48). You add a blank `.FORCE` target, and then add it as a dependency wherever you have `touch` now.